django/contrib/sessions/backends/file.py

```diff
@@ -57,13 +58,13 @@ def _key_to_file(self, session_key=None):
         if not set(session_key).issubset(VALID_KEY_CHARS):
             raise InvalidSessionKey("Invalid characters in session key")
 
-        return os.path.join(self.storage_path, self.file_prefix + session_key)
+        return self.storage_path / (self.file_prefix + session_key)
```

Comment:
Question: `Path` is generally slower (albeit not by much) than `os.path`. `_key_to_file` is called a lot around the class - should it be cached to prevent the need to keep constructing a `Path` instance?